__tests__/app.test.js

```diff
@@ -314,3 +312,29 @@ describe("DELETE: /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("200: responds with all users username, name, avatar_url", () => {
+    return request(app)
+      .get(`/api/users`)
+      .expect(200)
+      .then(({ body }) => {
+        const { users } = body;
+        expect(users.length).toBe(4);
+        users.forEach((user) => {
+          expect(typeof user.username).toBe("string");
+          expect(typeof user.name).toBe("string");
+          expect(typeof user.avatar_url).toBe("string");
+        });
+      });
+  });
+  test("404: responds with error message not found!", () => {
```

Comment:
This was most likely tested before in it's own describe block, where it should be, so you don't need this test here.